src/components/pages/AppMarketplace/components/AppListSection/index.tsx

```diff
@@ -147,49 +94,17 @@ export default function AppListSection() {
             },
             addButtonClicked: checkIsFavorite(card.id!),
           }))}
-          groupKey={group}
-        />
-      )
-    } else {
-      return (
-        <div style={{ textAlign: 'center' }}>
-          <CircularProgress
-            size={50}
-            sx={{
-              color: theme.palette.primary.main,
-            }}
-          />
-        </div>
-      )
-    }
-  }
-
-  return (
-    <Box ref={reference} className="overview-section">
-      <section className="overview-section-content">
-        <Typography
-          sx={{ fontFamily: 'LibreFranklin-Light' }}
-          variant="h3"
-          className="section-title"
-        >
-          {t('content.appstore.appOverviewSection.title')}
-        </Typography>
+        groupKey={control.group}
+      />
+    ) : (
```

Comment:
Is this else case necessary?
why do not we use 
```
{data && (
<Component />
)}
```

Reply:
@manojava-gk a React component always needs to return a JSX element so on the top level a 
`data && <....>`
would result in an undefined return value. So we return the empty JSX item here.